### PR TITLE
[US-3323] Update to twig 2.x

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|New feature| No
|BC breaks| Yes
|Fixes issues| No
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig [hunk 1]
```diff
@@ -4,10 +4,12 @@
 {% block h1 %}{{ 'Bestsellers'|trans }}{% endblock %}
 
 {% block main_content %}
+    {% import _self as self %}
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig]: does this kind of change belong into this commit?

File: packages/framework/src/Model/Feed/FeedXmlWriter.php [hunk 2]
```diff
@@ -68,18 +69,15 @@ public function writeItems(array $items, DomainConfig $domainConfig, $feedTempla
...
 
     /**
-     * @param \Twig_Template $twigTemplate
+     * @param \Twig_TemplateWrapper $twigTemplate
      * @param string $name
      * @param array $parameters
      * @return string
      */
-    private function getRenderedBlock(Twig_Template $twigTemplate, $name, array $parameters = [])
+    private function getRenderedBlock(Twig_TemplateWrapper $twigTemplate, $name, array $parameters = [])
```
Comment (vitek-rostislav) [inline packages/framework/src/Model/Feed/FeedXmlWriter.php]: it would be nice to rename `$twigTemplate` to `$twigTemplateWrapper`

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {% extends '@ShopsysFramework/Admin/Layout/layoutWithPanel.html.twig' %}
+{% import _self as self %}
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig]: This is little bit weird...I would appreciate at least link to some more information about this but I could not find any relevant :slightly_frowning_face: But maybe you could write a little bit more about it in the commit message - "when there is a macro defined in a template, it can not be used via '__self' in the same template anymore, it must be imported" or something like that

File: packages/framework/src/Resources/config/services_test.yml [hunk 4]
```diff
@@ -61,10 +61,20 @@ services:
 
   Shopsys\FrameworkBundle\Component\Domain\Config\DomainsConfigLoader: ~
 
+  Shopsys\FrameworkBundle\Component\Domain\Domain:
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/config/services_test.yml]: Why is this commit in this branch? It seems like it has nothing to do with the Twig update, am I right? Could you do that in a separate pull request with proper description please?

File: packages/framework/src/Model/Feed/FeedXmlWriter.php (hunk 2 above)
Comment (TomasVotruba) [reply packages/framework/src/Model/Feed/FeedXmlWriter.php]: @vitek-rostislav ...or you can cover these situations by a fixer: https://github.com/symplify/codingstandard#property-name-should-match-its-type-if-possible

File: packages/framework/src/Resources/config/services_test.yml (hunk 4 above)
Comment (TomasLudvik) [reply packages/framework/src/Resources/config/services_test.yml]: Removed

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig (hunk 3 above)
Comment (TomasLudvik) [reply packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig]: updated

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig (hunk 1 above)
Comment (TomasLudvik) [reply packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig]: No it does not :(
